dags/atd_metrobike_trips.py

```diff
@@ -0,0 +1,78 @@
+# Test locally with: docker compose run --rm airflow-cli dags test atd_metrobike_trips
+
+from datetime import datetime, timedelta
+
+from airflow.decorators import task
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from utils.slack_operator import task_fail_slack_alert
+
+default_args = {
+    "owner": "airflow",
+    "description": "Fetch metrobike trip data from dropbox and publish to Socrata",
+    "depend_on_past": False,
+    "start_date": datetime(2020, 12, 31),
+    "email_on_failure": False,
+    "email_on_retry": False,
```

Comment:
I don't think we're going to run with email as a mechanism to warn us when things go south, right? I think we're going to lean in to the slack notifier. If you agree, I think these can go.

Reply:
I made the same mistaken observation on Chia's PR! 🤦

This is right on and I'm going to start including these configuration directives.